pagopa/io-app
# feat(IT Wallet): [SIW-1789,SIW-1790] Always fetch the latest wallet instance status at app startup

## Short description
This PR calls the new wallet instance status endpoint at every app startup, without requesting an attestation. This removes the need to cache the attestation, thus solving the issue of an outdated wallet instance on the device.

## List of changes proposed in this pull request
- Call the new endpoint in place of the status attestation endpoint
- Modified tests

## How to test
Activate the wallet, then use IO Web to revoke it. Restart the app: the wallet should be instantly reset on the device.
